mapiproxy/services/plugins/dovecot/ChangeLog

```diff
@@ -0,0 +1 @@
+ * Removed openchange_cn option, always using email
```

Comment:
I disagree, this is dovecot plugin changelog, here we don't know at all proxyAddresses attribute.